render/detailed_node.go

```diff
@@ -79,6 +79,7 @@ func MakeDetailedNode(r report.Report, n RenderableNode) DetailedNode {
 	// multiple origins. The ultimate goal here is to generate tables to view
 	// in the UI, so we skip the intermediate representations, but we could
 	// add them later.
+	connections := []Row{}
 outer:
 	for _, id := range n.Origins {
 		if table, ok := OriginTable(r, id); ok {
```

Comment:
Should be able to get rid of the duplicates detection now (the only thing causing duplicates was origin endpoints)